.github/workflows/scripts/generate-weekly-report.js

```diff
@@ -181,6 +181,26 @@ async function getIssuesData({octokit, context}) {
   return stats
 }
 
+function generateComponentsLookingForOwnersReportSection(lookingForOwners) {
+  let count = 0
+  let section = []
+
+  for (const [componetType, components] of Object.entries(lookingForOwners)) {
+    count += Object.keys(components).length
+    // NOTE: the newline after <summary> is required for markdown to render correctly
+    section.push(`<details>
+    <summary> ${componetType}: ${Object.keys(components).length} </summary>\n`);
+
+    for (const [component, path] of Object.entries(components)) {
+      section.push(`- [ ] [${component}](../blob/master/${path}) `)
```

Comment:
I don't want to make this too complicated, but the current path is to the component's `metadata.yaml` file. I think it would be more user-friendly if we simply linked to the component. I believe we should be able to "simply" trim off `metadata.yaml` from the end of the URL and get the link we're looking for.

Do you think that's doable, or would it get too complicated here?

Reply:
change pushed, example of the new link here https://github.com/LucaLanziani/opentelemetry-collector-contrib/issues/7